client/state/invites/selectors.js

```diff
@@ -147,10 +148,11 @@ export function didInviteDeletionSucceed( state, siteId, inviteId ) {
  * @returns {boolean}          Whether an invite is being deleted
  */
 export function isDeletingAnyInvite( state, siteId ) {
-	return (
-		-1 !==
-		Object.values( get( state, [ 'invites', 'deleting', siteId ], {} ) ).indexOf( 'requesting' )
-	);
+	const invites = state.invites.deleting[ siteId ];
+	if ( ! invites ) {
+		return false;
+	}
+	return -1 !== Object.values( invites ).indexOf( 'requesting' );
```

Comment:
Could be shortened to:

```suggestion
	return Object.values( invites ).includes( 'requesting' );
```